packages/pulse-core/tests/state.test.ts

```diff
@@ -1,8 +1,245 @@
 import Pulse from '../lib';
 
 const App = new Pulse();
```

Comment:
We should always instantiate pulse with a `{ noCore: true }` configuration object when we are not using a core, so as to prevent unforeseen bugs.